pos_restrict_users/models/pos_config.py

```diff
@@ -0,0 +1,6 @@
+from odoo import models, fields
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+    assigned_user_ids = fields.Many2many("res.users", string="Assigned users")
```

Comment:
I would add a `help` attribute to this field, briefly explaining the mechanism. Not blocking though, just a suggestion

Reply:
Thanks, done